### DISCUSSION TITLE
Are there any plan for optimize the new offical support graphical card 7900xtx

### DISCUSSION THREAD

**Discussion by sdli1995** [ROCm/ROCm#2577] [Q&A] [answered]
It's very excited to hear that Rocm support 7900xtx offically. In my use case, i test serveral net works in difference precision and batchsize ,this card are not perform well against it's theroic performace in current rocm version 5.7.1 .
According the techup table ([https://www.techpowerup.com/gpu-specs/radeon-rx-7900-xtx.c3941]) 7900xtx has 61t fp32 flops and 120t fp16&bf16 flops and 960.0 GB/s memory bandwidth also have 96mb infinit cache, and we can see the rtx3090 which has 39t fp32 flops and 80t tensorcore fp16&bf16 flops and 936.0 GB/s memory bandwidth ,when i test lists network ,it performance not good 
here is my several network test both in 3090 and 7900xtx 

```
ref rtx3090

running benchmark for frameworks ['pytorch']
cuda version= 12.1
cudnn version= 8902
pytorch's batchsize at 16 vgg16 eval at fp32: 20.5ms avg
pytorch's batchsize at 16 vgg16 train at fp32: 58.7ms avg
pytorch's batchsize at 16 resnet152 eval at fp32: 27.9ms avg
pytorch's batchsize at 16 resnet152 train at fp32: 85.1ms avg
pytorch's batchsize at 16 densenet161 eval at fp32: 27.6ms avg
pytorch's batchsize at 16 densenet161 train at fp32: 87.4ms avg
pytorch's batchsize at 16 convnext_large eval at fp32: 57.3ms avg
pytorch's batchsize at 16 convnext_large train at fp32: 250.6ms avg
pytorch's batchsize at 16 swinbig eval at fp32: 39.9ms avg
pytorch's batchsize at 16 swinbig train at fp32: 124.2ms avg
pytorch's batchsize at 16 vgg16 eval at fp16: 12.4ms avg
pytorch's batchsize at 16 vgg16 train at fp16: 37.0ms avg
pytorch's batchsize at 16 resnet152 eval at fp16: 16.9ms avg
pytorch's batchsize at 16 resnet152 train at fp16: 63.1ms avg
pytorch's batchsize at 16 densenet161 eval at fp16: 20.4ms avg
pytorch's batchsize at 16 densenet161 train at fp16: 75.0ms avg
pytorch's batchsize at 16 convnext_large eval at fp16: 30.1ms avg
pytorch's batchsize at 16 convnext_large train at fp16: 87.6ms avg
pytorch's batchsize at 16 swinbig eval at fp16: 22.0ms avg
pytorch's batchsize at 16 swinbig train at fp16: 86.3ms avg
```

```
running benchmark for frameworks ['pytorch']
cuda version= None
cudnn version= 2020000
pytorch's batchsize at 16 vgg16 eval at fp32: 14.6ms avg
pytorch's batchsize at 16 vgg16 train at fp32: 71.2ms avg
pytorch's batchsize at 16 resnet152 eval at fp32: 23.2ms avg
pytorch's batchsize at 16 resnet152 train at fp32: 95.4ms avg
pytorch's batchsize at 16 densenet161 eval at fp32: 25.9ms avg
pytorch's batchsize at 16 densenet161 train at fp32: 100.3ms avg
pytorch's batchsize at 16 convnext_large eval at fp32: 108.3ms avg
pytorch's batchsize at 16 convnext_large train at fp32: 293.9ms avg
pytorch's batchsize at 16 swinbig eval at fp32: 43.3ms avg
pytorch's batchsize at 16 swinbig train at fp32: 142.4ms avg
pytorch's batchsize at 16 vgg16 eval at fp16: 8.8ms avg
pytorch's batchsize at 16 vgg16 train at fp16: 43.2ms avg
pytorch's batchsize at 16 resnet152 eval at fp16: 19.9ms avg
pytorch's batchsize at 16 resnet152 train at fp16: 72.5ms avg
pytorch's batchsize at 16 densenet161 eval at fp16: 22.2ms avg
pytorch's batchsize at 16 densenet161 train at fp16: 81.4ms avg
pytorch's batchsize at 16 convnext_large eval at fp16: 64.5ms avg
pytorch's batchsize at 16 convnext_large train at fp16: 168.2ms avg
pytorch's batchsize at 16 swinbig eval at fp16: 25.6ms avg
pytorch's batchsize at 16 swinbig train at fp16: 81.7ms avg
```

**Comment by briansp2020** [comment]
@sdli1995 
Can you post the source for the benchmark? I'd like to run it on my setup and compare also.
Thanks!

**Reply by sdli1995** [reply]
https://github.com/sdli1995/deep-learning-benchmark/tree/master here

**Reply by johnnynunez** [reply]
https://pypi.org/project/new-ai-benchmark/

**Reply by xvel** [reply]
Any news on that?

**Reply by veb-101** [reply]
Any updates?

**Comment by jamesxu2** [comment, marked as answer]
Hi @sdli1995 , thanks for providing the source of your benchmark. 

I'm not sure what your question is exactly. Can you clarify it? We are building ROCm to support the cards in our [official support matrix](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/reference/system-requirements.html#supported-gpus), which does include tuning for performance on those devices. 

I see you've quoted some benchmark stats - From your source, the 7900XTX has 61T fp32 flops while the 3090 has 39T fp32 flops. If you're implying that the ratio of model performance should reflect those benchmark numbers (61/39 ~ 1.5), I think there are more complicated factors involved that make a comparison like that infeasible. I have some comments in another discussion about using [benchmark results](https://github.com/ROCm/rocprofiler/issues/147#issuecomment-2451998676) in practical applications.

Anyways, I reran your test on a 7900XTX using a recent release of ROCm (6.2.4) and observe the following results, annotated next to your original results. **This is absolutely NOT an official AMD benchmark of any kind,** I just ran your benchmark locally to spare you from updating ROCm to latest and rerunning things yourself. My observation is that my results are very similar to what you measured.

I'm not sure what you can make of this, since the performance delta could probably be explained by system and configuration differences.

**Comment by schung-amd** [comment]
Closing as stale and unclear. We are continually improving ROCm on our supported GPUs, which includes the 7900XTX. Feel free to open another discussion topic or issue if you feel that performance on the 7900XTX or other AMD cards on current ROCm does not match your expectations.
